### PR TITLE
Fix for logits issue in Qwen3 VL MOE

### PR DESCRIPTION
**Pull request by tv-karthikeya**
Fix to support non layer wise export for N layers - updated logits to return lm.head() output instead of hidden_states(for layer wise)

### CODE REVIEW

**Comment by vbaddi** [comment]
duplicate #1047
